app/assets/constants.ts

```diff
@@ -0,0 +1,197 @@
+import { type IDropdownOption } from "../components/common/dropdown/dropdown.component";
+
+export enum Algorithms {
+  HS256 = "HS256",
+  HS384 = "HS384",
+  HS512 = "HS512",
+  ES256 = "ES256",
+  RS256 = "RS256",
+  PS256 = "PS256",
+}
+
+export interface IAlgorithmOption extends IDropdownOption {
+  signingMethodName: string;
+  requiresBothKeys?: boolean;
```

Comment:
a better boolean would be `isSymmetricKey` or something similar but you can leave it as is